app/views/layouts/application.html.erb

```diff
@@ -26,7 +26,7 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
   </head>
 
-  <body class="body_container">
+  <body class="body-container">
```

Comment:
not in keeping with syntax used in all other CSS styling rules - that being dashes between words rather than underscore.